.changeset/serious-houses-taste.md

```diff
@@ -1,9 +1,7 @@
 ---
-"rollup-config-forecast": patch
 "@trueplan/forecast-components": patch
 "@trueplan/forecast-icons": patch
 "@trueplan/forecast-theme": patch
-"@trueplan/forecast-website": patch
```

Comment:
Removing the website and rollup since we're not publishing those to NPM. This should fix the version package PR also.